core/input/mouse.js

```diff
@@ -108,27 +116,74 @@ Mouse.prototype = {
         this._handleMouseButton(e, 0);
     },
 
+    // Mouse wheel events are sent in steps over VNC. This means that the VNC
+    // protocol can't handle a wheel event with specific distance or speed.
+    // Therefor, if we get a lot of small mouse wheel events we combine them.
+    _generateWheelStepX: function (pos) {
```

Comment:
Perhaps we should keep the position stored on the object rather than passed along as an argument? In theory the mouse could have moved before the timer fires.

Reply:
Agreed, that feels more natural.